shell.nix

```diff
@@ -1,26 +1,18 @@
-{ pkgs ? import (fetchGit {
-    url = https://github.com/dapphub/dapptools.git;
-    rev = "6943c76bfb8e0b1fce54c3d9bba6f0f7e50d2f5c";
-  }) {}
-, mcd-cli ? pkgs.callPackage (import (fetchGit {
-    url = https://github.com/makerdao/mcd-cli.git;
-    rev = "86842b49defa53301ac0019f7d5994859bb3e1e9";
-  })) {}
-, sethret ? (import (fetchGit {
-    url = https://github.com/icetan/sethret.git;
-    rev = "ef77915e2881011603491275f36b44bf2478b408";
-  }) {}).sethret
-}:
+{ pkgsSrc ? (import ./nix/pkgs.nix {}).pkgsSrc
+, pkgs ? (import ./nix/pkgs.nix { inherit pkgsSrc; }).pkgs
+}: with pkgs;
 
-pkgs.mkShell {
-  buildInputs = with pkgs; [
-    dapp ethsign seth mcd-cli sethret
-    bc jq coreutils findutils procps
+let
+  tdds = import ./. { inherit pkgs; };
+in mkShell {
+  buildInputs = tdds.bins ++ [
+    tdds
```

Comment:
Hmm... So you need to add this package's derivation as one of the `buildInputs` in its `shell.nix`? Kinda inception-worthy :D

Reply:
Yes, if you want the output of `tdds` (i.e. `*-deploy` bins) to be available inside the shell.